_posts/2016-03-28-chime-quality-assurance.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  "QA Automation on Chime"
+date:   2016-03-28 11:00:00
+categories: qa
+author: "Nick Fagen"
+---
+![Sittercity QA](/assets/sittercity_qa_logo.png)
+
+So far, 2016 has been a great year. Our small, but rockstar, QA team has been making huge improvements towards improving automated regression suites across all of our projects. We have tackled the endless QA dilemma regarding automation, how to do it effectively and efficiently. There are always obstacles when automating any app. For example, let's say you have a new release and bug fixes ready to ship. How will you ensure that the new bug fix(es) have not introduced any new issues in previous working functionality? Manually testing all the functionality every single time you have some bug fixes or new feature addition is an option when a project is just starting.  However, as a project grows, large manual test regression plans are daunting and in my opinion a red flag. With larger regression plans you might be able to accomplish it, but then you are not going to be testing effectively in terms of company cost, resources, time, etc. More and more companies are trying to move toward continuous delivery, making a need for QA automation.
```

Comment:
Can we reword the following a bit? Maybe just leave out the 'have some bug fixes', like this:

```Manually testing all of the functionality of an application for every single fix or new feature is an option when a project is just starting.
```